keras/models.py

```diff
@@ -137,7 +137,9 @@ def get_json_type(obj):
                 weight_values = K.batch_get_value(symbolic_weights)
                 weight_names = []
                 for i, (w, val) in enumerate(zip(symbolic_weights, weight_values)):
-                    if hasattr(w, 'name') and w.name:
+
+                    # Default values of symbolic_weights /variable
+                    if hasattr(w, 'name') and w.name != "/variable":
```

Comment:
Make sense. I have added backend check. Test cases pass for both tensorflow and theano.